lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -117,6 +117,10 @@ def log_fail_fast_limit_met
 
         private
 
+        def post_trap_signals
+          RSpec.world.wants_to_quit = true
```

Comment:
@3v0k4 I'm thinking about adding the following seed here so that we could help with [the issue mentioned here](https://github.com/KnapsackPro/knapsack_pro-ruby/issues/285#issuecomment-2514693380). What do you think?

```suggestion
          puts "Randomized with seed #{@rspec_runner.knapsack__seed.value}"
          RSpec.world.wants_to_quit = true
```

The output would look like this when a signal is received:

```
^CINT signal has been received. Terminating Knapsack Pro...
Randomized with seed 44874

================================================================================
Start logging 3 detected threads.
Use the following backtrace(s) to find the line of code that got stuck if the CI node hung and terminated your tests.
How to read the backtrace: https://knapsackpro.com/perma/ruby/backtrace-debugging

Main thread backtrace:
...
```

Reply:
Doesn't RSpec already log the seed?

Reply:
Also, I'd expect this to happen in a separate PR.